topics/dev/tutorials/tool-integration/slides.html

```diff
@@ -585,7 +585,7 @@
 $ planemo serve
 ```
 
-Open http://127.0.0.1:9090/ in your web browser to view your new tool
+Open https://127.0.0.1:9090/ in your web browser to view your new tool
```

Comment:
thanks! changed all the localhost links back, but forgot about the 127.0.0.1, nice catch again :)